deploy/playbooks/examples/deploy_vagrant.yml

```diff
@@ -1,33 +1,17 @@
 ---
-
-# install python2.7 on xenial nodes
```

Comment:
The playbook only supports Xenial so I just moved the package to `system_packages`